### PR TITLE
global: migration from bower to npm

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: invenio_theme/bundles.py [hunk 1]
```diff
@@ -36,25 +36,29 @@
...
-        'bower_components/almond/almond.js',
+    NpmBundle(
+        'node_modules/almond/almond.js',
         'js/settings.js',
         filters='uglifyjs',
-        bower={"almond": "~0.3.1", }
+        npm={
+            "almond": "~0.3.1",
+            "jquery": "~1.9.1"
+        }
```
Comment (greut) [inline invenio_theme/bundles.py]: This one is not picked up, right?

Comment (drjova) [inline invenio_theme/bundles.py]: yep

Comment (greut) [inline invenio_theme/bundles.py]: @drjova I've got a fix for that. PR incoming ;-)

Comment (drjova) [inline invenio_theme/bundles.py]: :dancer:

File: examples/app.py [hunk 2]
```diff
@@ -24,20 +24,19 @@
 
 """Minimal Flask application for development.
 
-Installing bower requirements
+Installing npm requirements
 -----------------------------
```
Comment (jirikuncar) [inline examples/app.py]: remove 2 `-`

**Comment by drjova** [comment]
@jirikuncar done :+1:

**Comment by jirikuncar** [comment]
@drjova Invenio-Assets v1.0.0a3 is out. So you can update `setup.py`.

**Comment by drjova** [comment]
@jirikuncar done!

**Pull request merged**